frontend/app/src/scenes/Home/scenes/AddCategory/index.tsx

```diff
@@ -59,34 +104,44 @@ class AddCategory extends React.Component<
       <div>
         <Row className={mainStyles.innerContent}>
           <Col lg={24} xl={{ span: 20, offset: 2 }}>
-            <Title>Add a new Category</Title>
-            <Row>
-              <Col md={12}>
-                <Form size={'large'} onFinish={this.onFinish}>
-                  <Spin spinning={this.state.isLoading}>
-                    <Title level={3}>Name</Title>
-                    <Text>Name of the Category</Text>
-                    <Form.Item
-                      name={'name'}
-                      className={mainStyles.formItem}
-                      rules={[
-                        {
-                          required: true,
-                          message: 'Please enter a category name!',
-                        },
-                      ]}
-                    >
-                      <Input placeholder="ex: KG & Primary" />
-                    </Form.Item>
-                    <Form.Item>
-                      <Button type="primary" htmlType="submit">
-                        Add Category
-                      </Button>
-                    </Form.Item>
-                  </Spin>
-                </Form>
-              </Col>
-            </Row>
+            {this.state.isTypeEdit ? (
+              <Title level={2}>Update Category: {this.categoryId}</Title>
```

Comment:
Let's remove the category ID from the ui

Reply:
Sure, On it.

Reply:
Resolved with commit [1628960](https://github.com/sef-global/sef-core/pull/81/commits/162896037fd9f9a8c5ea86a786ce67869de5071b)